### PR TITLE
Updating some things

### PR DESCRIPTION
**Pull request by Jolly-66**
Planned things:

Say Sounds
- [x] Felinid 
- [x] Shadow person

Bridge Officer
 - [x] Update the locker sprite
 - [x] Add a map turf for it
 - [x] Update their ID and HUD icons

 Misc
 - [x] Add landmark icons for BO, Toxi and Xenobio (that aren't borrowed)
 - [x] Update scientist hud icon

### CODE REVIEW

File: jollystation_modules/code/game/objects/structures/crate_lockers/closets/secure/bridge_officer.dm [hunk 1]
```diff
@@ -27,7 +27,9 @@
 /obj/structure/closet/secure_closet/bridge_officer
 	name = "\proper bridge officer's locker"
 	req_access = list(ACCESS_HEADS)
-	icon_state = "hos"
+	icon = 'jollystation_modules/icons/obj/locker.dmi'
+	icon_state = "bo"
+	icon_door = "bo"
```
Comment (MrMelbert) [inline jollystation_modules/code/game/objects/structures/crate_lockers/closets/secure/bridge_officer.dm:32]:
```suggestion
```
this should work exactly the same without needing to set this

File: code/datums/id_trim/jobs.dm [hunk 2]
```diff
@@ -410,6 +410,7 @@
 
 /datum/id_trim/job/scientist
 	assignment = "Scientist"
+	trim_icon = 'jollystation_modules/icons/obj/card.dmi' //NON-MODULE CHANGE
```
Comment (MrMelbert) [inline code/datums/id_trim/jobs.dm:413]: wait if this works, and you can swap back and forth between non-modular and modular trims without any errors, then we can get rid of all of the hacky stuff in `cards_ids.dm`

File: jollystation_modules/code/modules/mob/living/carbon/human/species_types/shadowpeople.dm [hunk 3]
```diff
@@ -1,6 +1,6 @@
 // -- Shadowperson (& Nightmare) species additions --
 /datum/species/shadow
-	species_speech_sounds = list('jollystation_modules/sound/voice/shadow1.ogg' = 70, \
-								'jollystation_modules/sound/voice/shadow2.ogg' = 60)
-	species_speech_sounds_ask = list('jollystation_modules/sound/voice/shadow_ask.ogg' = 80)
-	species_speech_sounds_exclaim = list('jollystation_modules/sound/voice/shadow_exclaim.ogg' = 80)
+	species_speech_sounds = list('jollystation_modules/sound/voice/shad1.ogg' = 55, \
+								'jollystation_modules/sound/voice/shad_exclaim.ogg' = 55)
+	species_speech_sounds_exclaim = list('jollystation_modules/sound/voice/shad2.ogg' = 55)
+	//I'm unable to swap the names of shad2 and shad_exclaim due to issues with the .ogg type on my PC - Jolly
```
Comment (MrMelbert) [inline jollystation_modules/code/modules/mob/living/carbon/human/species_types/shadowpeople.dm:6]: meme

Comment (Jolly-66) [reply jollystation_modules/code/modules/mob/living/carbon/human/species_types/shadowpeople.dm:6]: i seriously think vscode is fucking with the .ogg file type on my end

**Comment by Jolly-66** [comment]
should be done, CI shouldn't fail, hopefully.

**Review by MrMelbert — APPROVED** [review]

**Pull request merged**